hazelcast/connection.py

```diff
@@ -361,6 +371,16 @@ def _translate(self, address):
 
         return translated
 
+    def _translate_member_address(self, member):
+        if self._use_public_addresses:
+            public_address = member.address_map.get(_CLIENT_PUBLIC_ENDPOINT_QUALIFIER, None)
```

Comment:
we don't need to specify default None

Reply:
I know, but, I like being explicit here.  If you look at the client codebase, you would see that there is almost always an explicit default value